common/metadata-helper.cpp

```diff
@@ -240,13 +260,14 @@ namespace rs2
                         std::wstring metadatakey = ss.str();
 
                         DWORD MetadataBufferSizeInKB = 0;
-                        RegQueryValueEx(
+                        if (RegQueryValueEx(
                             key,
                             metadatakey.c_str(),
                             NULL,
                             NULL,
                             (LPBYTE)(&MetadataBufferSizeInKB),
-                            &len);
+                            &len) != ERROR_SUCCESS)
+                            rs2::log(RS2_LOG_SEVERITY_DEBUG, "Unable to read metadata registry key!");
```

Comment:
Might want to include the key path in the message, otherwise it lacks context and will be repetitive if it happens multiple times